### DISCUSSION TITLE
when I refresh page, audio did not played

### DISCUSSION THREAD

**Discussion by almaz1999kgz** [katspaugh/wavesurfer.js#2865] [Q&A]
I created wavesurfer 7 beta version. Here you appear to have removed functions such as backend, audiocontext, etc. So I connected to Web Audio to set it up. Further, when I set everything up and launched everything works, but when I refresh the page, the music does not play, sorry for the mistakes, I'm not English-speaking. help me please. My code
onMounted( () => {
      audio.value = new Audio()
      audio.value.controls = true
      audio.value.src = files.value
      wavesurfer.value = WaveSurfer.create({
        container: "#" + waveform,
        cursorColor: "#e2007a",
        audioRate: parseFloat(speed.value),
        removeMediaElementOnDestroy: true,
        height: 80,
        responsive: true,
        media: audio.value,
        cursorWidth: 2,
        mediaType: "audio",
        barHeight: 0.5,
        splitChannels:[
          {
          waveColor: '#bababa',
          progressColor: '#757575',
        },
          {
            waveColor: '#e2007a',
            progressColor: '#a4035a',
          },
        ]
      });

const context = new AudioContext();
      const audioNode = context.createMediaElementSource(audio.value);

      const splitter = context.createChannelSplitter(2);
      const merger = context.createChannelMerger(2);

      const input = context.createGain();
      input.channelCountMode = "explicit";
      merger.connect(context.destination);

      gainL.value = context.createGain();
      gainR.value = context.createGain();

      audioNode.connect(splitter);

      splitter.connect(gainL.value, 0, 0);
      splitter.connect(gainR.value, 1, 0);

      gainL.value.connect(merger, 0, 0);
      gainR.value.connect(merger);
      gainR.value.connect(merger, 0, 1);
      gainL.value.connect(merger, 0, 1)

**Comment by almaz1999kgz** [comment]
Or is it possible instead of using "media". Use "load" but make settings (webAudio) remain

**Reply by katspaugh** [reply]
Your code looks correct.

**Reply by almaz1999kgz** [reply]
you have a bug, go to the address and click play after refresh the page and click the play button again, you will see for yourself
https://wavesurfer.pages.dev/examples/#webaudio.js
